### PR TITLE
REFACTOR-#2832: removed limitation on the micro part of python version in CI

### LINKED ISSUE #2832: REFACTOR: remove limitation on the micro part of python version in CI
This needs to be done for consistency.

### PR DESCRIPTION
**Pull request by anmyachev**
Signed-off-by: Anatoly Myachev <anatoly.myachev@intel.com>

## What do these changes do?

- [ ] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [ ] passes `flake8 modin`
- [ ] passes `black --check modin`
- [ ] signed commit with `git commit -s` 
- [x] Resolves #2832 
- [ ] tests added and passing

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -274,7 +274,7 @@ jobs:
         uses: conda-incubator/setup-miniconda@v2
         with:
           activate-environment: modin_on_omnisci
-          python-version: 3.7.8
+          python-version: 3.7
```
Comment (devin-petersohn) [inline .github/workflows/ci.yml:277]: The minimum version for pandas is 3.7.1
```suggestion
          python-version: 3.7.1
```

Comment (anmyachev) [reply .github/workflows/ci.yml:277]: When we define python version like that: `python-version: 3.7` the latest version available on the channel is installed. At the moment it is `3.7.10`.

Comment (devin-petersohn) [reply .github/workflows/ci.yml:277]: Sounds good

**Review by devin-petersohn — DISMISSED** [review]

**Review by devin-petersohn — APPROVED** [review]
I see the title of the PR changed, but commit message stayed. I will move the title to the message.

**Pull request merged**